adaptive/src/main/java/com/google/accompanist/adaptive/TwoPane.kt

```diff
@@ -0,0 +1,411 @@
+    ),
+    windowGeometry = windowGeometry
+)
+
+/**
+ * Returns a [TwoPaneStrategy] that will place the slots horizontally if there is a vertical fold.
+ *
+ * If there is no horizontal fold, then the [fallbackStrategy] will be used instead.
+ */
+public fun HorizontalTwoPaneStrategy(
+    fallbackStrategy: TwoPaneStrategy,
+    windowGeometry: WindowGeometry,
+): TwoPaneStrategy = TwoPaneStrategy { density, layoutDirection, layoutCoordinates ->
+    val verticalFold = windowGeometry.displayFeatures.find {
+        it is FoldingFeature && it.orientation == FoldingFeature.Orientation.VERTICAL
+    } as FoldingFeature?
+
+    if (verticalFold != null &&
+        (
+            verticalFold.isSeparating ||
+                verticalFold.occlusionType == FoldingFeature.OcclusionType.FULL
+            ) &&
+        verticalFold.bounds.toComposeRect().overlaps(layoutCoordinates.boundsInWindow())
+    ) {
+        val foldBounds = verticalFold.bounds.toComposeRect()
+        SplitResult(
+            isHorizontalSplit = true,
+            splitArea = Rect(
+                layoutCoordinates.windowToLocal(foldBounds.topLeft),
+                layoutCoordinates.windowToLocal(foldBounds.bottomRight)
+            )
+        )
+    } else {
+        fallbackStrategy.calculateSplitResult(density, layoutDirection, layoutCoordinates)
+    }
+}
+
+/**
+ * Returns a [TwoPaneStrategy] that will place the slots vertically if there is a horizontal fold.
+ *
+ * If there is no vertical fold, then the [fallbackStrategy] will be used instead.
+ */
+public fun VerticalTwoPaneStrategy(
+    fallbackStrategy: TwoPaneStrategy,
+    windowGeometry: WindowGeometry,
+): TwoPaneStrategy = TwoPaneStrategy { density, layoutDirection, layoutCoordinates ->
+    val horizontalFold = windowGeometry.displayFeatures.find {
+        it is FoldingFeature && it.orientation == FoldingFeature.Orientation.HORIZONTAL
+    } as FoldingFeature?
+
+    if (horizontalFold != null &&
+        (
+            horizontalFold.isSeparating ||
+                horizontalFold.occlusionType == FoldingFeature.OcclusionType.FULL
+            ) &&
+        horizontalFold.bounds.toComposeRect().overlaps(layoutCoordinates.boundsInWindow())
+    ) {
+        val foldBounds = horizontalFold.bounds.toComposeRect()
+        SplitResult(
+            isHorizontalSplit = false,
+            splitArea = Rect(
+                layoutCoordinates.windowToLocal(foldBounds.topLeft),
+                layoutCoordinates.windowToLocal(foldBounds.bottomRight)
+            )
+        )
+    } else {
+        fallbackStrategy.calculateSplitResult(density, layoutDirection, layoutCoordinates)
+    }
+}
+
+/**
+ * Returns a [TwoPaneStrategy] that will place the slots horizontally.
+ *
+ * The split will be placed at the given [splitFraction] from start, with the given [splitWidth].
+ *
+ * This strategy is _not_ fold aware.
+ */
+public fun FractionHorizontalTwoPaneStrategy(
+    splitFraction: Float,
```

Comment:
Agreed, updated "split" to always be used as a verb, and "gap" to refer to the thing created by splitting.

I also renamed to `gapBounds` instead of `gapArea`, I think that's a little bit more inline from some of the other APIs.